src/exercises/assignBookmarksToExercises.js

```diff
@@ -0,0 +1,101 @@
+import {NUMBER_OF_BOOKMARKS_TO_PRACTICE} from "./exerciseSequenceTypes";
```

Comment:
@DarinkaGM  - or this added file for example. why does it appear as newly added? It is already in development! 

Maybe we should meet on Monday, and do another PR that only contains your changes?